libraries/c_sdk/standard/mqtt/test/access/iot_test_access_mqtt.h

```diff
@@ -63,17 +63,17 @@ _mqttConnection_t * IotTestMqtt_createMqttConnection( bool awsIotMqttMode,
 /*----------------------- iot_mqtt_subscription.c -----------------------*/
 
 /* Internal data structures of iot_mqtt_subscription.c, redefined for the tests. */
-typedef struct _topicMatchParams
+/*typedef struct _topicMatchParams
 {
     const char * pTopicName;
     uint16_t topicNameLength;
     bool exactMatchOnly;
-} _topicMatchParams_t;
-typedef struct _packetMatchParams
+} _topicMatchParams_t;*/
+/*typedef struct _packetMatchParams
 {
     uint16_t packetIdentifier;
     int32_t order;
-} _packetMatchParams_t;
+} _packetMatchParams_t;*/
```

Comment:
Why are these kept in comments? Delete them if not required

Reply:
Addressed